app/lib/location_sessions_factory.rb

```diff
@@ -8,6 +8,7 @@ def initialize(location, academic_year:, sync_patient_teams_now: false)
   end
 
   def call
+    imported_patient_location_ids = []
     ActiveRecord::Base.transaction do
```

Comment:
`transaction` automatically returns the last result of the block, so you can do something like this:

```rb
result = ActiveRecord::Base.transaction do
  add_patients!
end

imported_patient_location_ids = result.ids
```

This would be a more typical Ruby style.